conf/etc/v2/event_analyzer.service

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+[Unit]
+Description=HA event analyzer daemon process
+After=syslog.target
+
+[Service]
+Type=simple
+ExecStartPre=/usr/bin/sleep 5
+ExecStart=/usr/bin/event_analyzer_d
+ExecStopPre=/usr/bin/sleep 10
+TimeoutStopSec=7sec
```

Comment:
How did we decide on this and above two sleep values?

Reply:
Do we need it can we remove ?
i think default is 90 sec and after that it send kill signal to process

Reply:
> How did we decide on this and above two sleep values?
No discussion as such was done for these timings.
The sleeps are the time to wait before executing the respective ExecStart/stop command 
we can change these if other values make more sense.

Reply:
>Do we need it can we remove ?
Yes we can remove the timeout if default is fine with all

Reply:
@indrajitzagade please comment if changes are needed

Reply:
I think we can tune these going forward.